frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/EnvironmentHeader.tsx

```diff
@@ -56,7 +54,12 @@ export const EnvironmentHeader: FC<EnvironmentHeaderProps> = ({
         <StyledAccordionSummary
             {...props}
             expandIcon={
-                expandable ? <ExpandMore titleAccess='Toggle' /> : <></>
+                <ExpandMore
+                    titleAccess='Toggle'
+                    sx={{
+                        opacity: expandable ? 1 : 0,
```

Comment:
In case that's what you're doing here: opacity doesn't affect whether something is reachable by tab, but that might be handled in a different manner.

Reply:
Handled with `visibility` + `aria-hidden`